.pre-commit-config.yaml

```diff
@@ -41,6 +41,6 @@ repos:
         args: ["--filter-files"]
 
   - repo: https://github.com/psf/black
-    rev: 24.1.1
+    rev: 23.12.1
```

Comment:
why you have degraded version?

Reply:
There seem to be contradicting issues here, I am making the release through for now